server/db/seedscript.js

```diff
@@ -0,0 +1,83 @@
+const faker = require('faker');
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/gameinfo', {useNewUrlParser: true, useUnifiedTopology: true});
+const db = mongoose.connection;
+
+
+const seedingScript = new mongoose.Schema({
```

Comment:
This all looks really good! You've definitely shown you're able to use a lot of different tools to accomplish seeding your DB (async/await, aspects of mongo schema-ing that I didn't even know were possible until now). My only piece of feedback would be to rename your schema to something other than "seedingscript", perhaps something like "game_info_carousel_data" or something that describes whats in the schema - that way when it exists amongst other schemas/data, someone looking at the schema name will be able to tell what's in your schema without having to run a query into it. Overall, great work though!